Library/Formula/kibana.rb

```diff
@@ -0,0 +1,71 @@
+class Kibana < Formula
+  homepage 'https://www.elastic.co/products/kibana'
+  url 'https://download.elastic.co/kibana/kibana/kibana-4.0.2-darwin-x64.tar.gz'
+  version '4.0.2'
+  sha1 'ce6136d826cf8ac1da71abea419f2f5220055933'
+  head 'https://github.com/elastic/kibana.git'
+
+  def install
+    # install base files
+    prefix.install %w[ node src ]
+
+    # setup etc path
+    etc_path.mkpath
+
+    # setup var path
+    var_path.mkpath
```

Comment:
I like the def blocks so that I don't have to repeat path definitions in multiple places. Makes it easier to change later if the package changes in the future